vm/systemSmartContracts/delegation.go

```diff
@@ -26,6 +26,7 @@ const serviceFeeKey = "serviceFee"
 const totalActiveKey = "totalActive"
 const rewardKeyPrefix = "reward"
 const fundKeyPrefix = "fund"
+const maxUnStakedFunds = 50
```

Comment:
this is an actual len, not the funds.

Reply:
maxNumOfUnStakes or maxNumOfUnStakedFunds ?

Reply:
pushed